src/mcts/search.cc

```diff
@@ -2431,6 +2441,12 @@ void SearchWorker::DoBackupUpdateSingleNode(
       m = n->GetM();
     }
     n->FinalizeScoreUpdate(v, d, m, node_to_process.multivisit);
+    if (params_.GetUpdateInterval() > 0 &&
+        (n->GetNStarted() + 1) % params_.GetUpdateInterval() == 0) {
```

Comment:
the +1 here is to make sure that it takes a full interval until the first update.